service-registry-client/src/main/java/no/difi/meldingsutveksling/serviceregistry/SRParameter.java

```diff
@@ -38,6 +38,37 @@ public String getQuery() {
         return query.toString();
     }
 
+    public String getUrlTemplate() {
+        if (infoOnly) {
+            return "info/{identifier}";
+        }
+        StringBuilder urlTemplateBuilder = new StringBuilder();
+        urlTemplateBuilder.append("identifier/{identifier}?{queryString}");
+        if (! isNullOrEmpty(process)) {
+            urlTemplateBuilder.append(("/process/{process}?{queryString}"));
+        }
+        return urlTemplateBuilder.toString();
+    }
+
+    public Map<String, String> getUrlVariables() {
+        Map<String, String> urlVariables = new HashMap<>();
+        urlVariables.put("identifier", identifier);
+        if (infoOnly) {
+            return urlVariables;
+        }
+        if (! isNullOrEmpty(process)) {
+            urlVariables.put("process", process);
+        }
+        if (securityLevel != null) {
+            urlVariables.put("securityLevel", String.valueOf(securityLevel));
+        }
+        if (! isNullOrEmpty(conversationId)) {
```

Comment:
Bør merge inn siste development for å få med print endringane, dei ser ut til å verte påverka av denne changen.

Reply:
Bra sett, har lagt inn